### DISCUSSION TITLE
How to change default i18n and use one language?

### DISCUSSION THREAD

**Discussion by mrvectopus** [shuding/nextra#2782] [Q&A]
Hey 👨‍💻 

I have problem with Search when trying change a default language and use only one language.

When I set only one language in i18n config (to change default meta tag `html lang="en"`), I'm get error in search results:

```
TypeError: undefined is not iterable (cannot read property Symbol(Symbol.iterator))
```

`next.config.js`

```
i18n: {
    locales: ['de'],
    defaultLocale: 'de',
    localeDetection: false,
  }
```

`middleware.js`
```
export { locales as middleware } from 'nextra/locales'
```

`.mdx` pages and `_meta.json` without language prefix `de`

I was read docs about a language prefix, but this is required only if we have multi language pages, I don't need multi language, I need to change only default language...

**Comment by Srinath10X** [comment]
Once can you check 

1.the definition of the locales variable in your code
2. Verify that the locales variable is being imported correctly in your middleware.js file and imported correctly
3. Double-check the configuration in your next.config.js file related to i18n and locales.
4. Look for any other potential causes
